Elecritic/Pages/ProductPage.razor.cs

```diff
@@ -53,16 +79,11 @@ public class ReviewDto {
 
             public ReviewDto() { }
 
-            public ReviewDto(string reviewText, int ratingProduct, string recommended) {
-                Text = reviewText;
-                RatingProduct = ratingProduct;
-                Recommended = recommended;
-            }
-
             /// <summary>
             /// Simply sets everything as empty or as 0
             /// </summary>
-            public void ClearReview() {
+            public void Clear() {
+                Title = "";
                 Text = "";
                 RatingProduct = 0;
                 Recommended = "";
```

Comment:
Okay, title and now the void is called 'Clear', I guess it was redundant hmm

Reply:
> 
> 
> Okay, title and now the void is called 'Clear', I guess it was redundant hmm

yeah I renamed it because once I used the method I thought `ReviewDto.Clear()` is clearer to read